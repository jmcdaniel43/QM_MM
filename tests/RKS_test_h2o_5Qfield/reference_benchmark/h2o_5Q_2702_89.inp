molecule h2o {
O	30.275	30.306	30.18
H	29.677	30.07	30.945
H	30.855	29.523	29.956
symmetry c1
no_reorient
no_com
}

Chrgfield = QMMM()
#  q , x , y , z
Chrgfield.extern.addCharge(2.0, 33.0 , 30.0 , 30.0)
Chrgfield.extern.addCharge(-1.0, 25.9 , 25.0 , 30.0)
Chrgfield.extern.addCharge(-1.0, 36.1 , 35.0 , 30.0)
Chrgfield.extern.addCharge( 1.0, 45.9 , 30.0 , 30.0)
Chrgfield.extern.addCharge(-1.0, 20.1 , 35.0 , 30.0)
psi4.set_global_option_python('EXTERN', Chrgfield.extern)

set {
basis aug-cc-pvdz
dft_spherical_points 2702
dft_radial_points 89
scf_type df
}

energy('PBE')

