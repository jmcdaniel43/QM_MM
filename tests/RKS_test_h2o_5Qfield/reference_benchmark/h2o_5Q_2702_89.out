
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 1.4a2.dev35 

                         Git: Rev {master} 03b714c dirty


    R. M. Parrish, L. A. Burns, D. G. A. Smith, A. C. Simmonett,
    A. E. DePrince III, E. G. Hohenstein, U. Bozkaya, A. Yu. Sokolov,
    R. Di Remigio, R. M. Richard, J. F. Gonthier, A. M. James,
    H. R. McAlexander, A. Kumar, M. Saitow, X. Wang, B. P. Pritchard,
    P. Verma, H. F. Schaefer III, K. Patkowski, R. A. King, E. F. Valeev,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, and C. D. Sherrill,
    J. Chem. Theory Comput. 13(7) pp 3185--3197 (2017).
    (doi: 10.1021/acs.jctc.7b00174)


                         Additional Contributions by
    P. Kraus, H. Kruse, M. H. Lechner, M. C. Schieber, R. A. Shaw,
    A. Alenaizan, R. Galvelis, Z. L. Glick, S. Lehtola, and J. P. Misiewicz

    -----------------------------------------------------------------------


    Psi4 started on: Monday, 27 January 2020 08:41AM

    Process ID: 29744
    Host:       rich133-k41-16-l.pace.gatech.edu
    PSIDATADIR: /nv/hp22/jmcdaniel43/data/Programs/psi4/psi4/bin/share/psi4
    Memory:     500.0 MiB
    Threads:    1
    
  ==> Input File <==

--------------------------------------------------------------------------
molecule h2o {
O	30.275	30.306	30.18
H	29.677	30.07	30.945
H	30.855	29.523	29.956
symmetry c1
no_reorient
no_com
}

Chrgfield = QMMM()
#  q , x , y , z
Chrgfield.extern.addCharge(2.0, 33.0 , 30.0 , 30.0)
Chrgfield.extern.addCharge(-1.0, 25.9 , 25.0 , 30.0)
Chrgfield.extern.addCharge(-1.0, 36.1 , 35.0 , 30.0)
Chrgfield.extern.addCharge( 1.0, 45.9 , 30.0 , 30.0)
Chrgfield.extern.addCharge(-1.0, 20.1 , 35.0 , 30.0)
psi4.set_global_option_python('EXTERN', Chrgfield.extern)

set {
basis aug-cc-pvdz
dft_spherical_points 2702
dft_radial_points 89
scf_type df
}

energy('PBE')

--------------------------------------------------------------------------

*** tstart() called on rich133-k41-16-l.pace.gatech.edu
*** at Mon Jan 27 08:41:35 2020

   => Loading Basis Set <=

    Name: AUG-CC-PVDZ
    Role: ORBITAL
    Keyword: BASIS
    atoms 1   entry O          line   254 file /nv/hp22/jmcdaniel43/data/Programs/psi4/psi4/bin/share/psi4/basis/aug-cc-pvdz.gbs 
    atoms 2-3 entry H          line    40 file /nv/hp22/jmcdaniel43/data/Programs/psi4/psi4/bin/share/psi4/basis/aug-cc-pvdz.gbs 


         ---------------------------------------------------------
                                   SCF
               by Justin Turney, Rob Parrish, Andy Simmonett
                          and Daniel G. A. Smith
                              RKS Reference
                        1 Threads,    500 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
         O           30.275000000000    30.306000000000    30.180000000000    15.994914619570
         H           29.677000000000    30.070000000000    30.945000000000     1.007825032230
         H           30.855000000000    29.523000000000    29.956000000000     1.007825032230

  Running in c1 symmetry.

  Rotational constants: A =      8.96367  B =      0.00034  C =      0.00034 [cm^-1]
  Rotational constants: A = 268724.01754  B =     10.22522  C =     10.22506 [MHz]
  Nuclear repulsion =    8.794824250261485

  Charge       = 0
  Multiplicity = 1
  Electrons    = 10
  Nalpha       = 5
  Nbeta        = 5

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is SAD.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: AUG-CC-PVDZ
    Blend: AUG-CC-PVDZ
    Number of shells: 19
    Number of basis function: 41
    Number of Cartesian functions: 43
    Spherical Harmonics?: true
    Max angular momentum: 2

  ==> DFT Potential <==

   => Composite Functional: PBE <= 

    PBE GGA Exchange-Correlation Functional

    J.P. Perdew et. al., Phys. Rev. Lett., 77(18), 3865-3868, 1996


    Deriv               =              1
    GGA                 =           TRUE
    Meta                =          FALSE

    Exchange Hybrid     =          FALSE
    MP2 Hybrid          =          FALSE

   => Exchange Functionals <=

    1.0000     XC_GGA_X_PBE

   => Correlation Functionals <=

    1.0000     XC_GGA_C_PBE

   => Molecular Quadrature <=

    Radial Scheme          =       TREUTLER
    Pruning Scheme         =           NONE
    Nuclear Scheme         =       TREUTLER

    BS radius alpha        =              1
    Pruning alpha          =              1
    Radial Points          =             89
    Spherical Points       =           2702
    Total Points           =         703393
    Total Blocks           =           4768
    Max Points             =            256
    Max Functions          =             41
    Weights Tolerance      =       1.00E-15

   => Loading Basis Set <=

    Name: (AUG-CC-PVDZ AUX)
    Role: JKFIT
    Keyword: DF_BASIS_SCF
    atoms 1   entry O          line   270 file /nv/hp22/jmcdaniel43/data/Programs/psi4/psi4/bin/share/psi4/basis/aug-cc-pvdz-jkfit.gbs 
    atoms 2-3 entry H          line    70 file /nv/hp22/jmcdaniel43/data/Programs/psi4/psi4/bin/share/psi4/basis/aug-cc-pvdz-jkfit.gbs 

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A         41      41       0       0       0       0
   -------------------------------------------------------
    Total      41      41       5       5       5       0
   -------------------------------------------------------

  ==> Integral Setup <==

  DFHelper Memory: AOs need 0.002 GiB; user supplied 0.002 GiB. Using in-core AOs.

  ==> MemDFJK: Density-Fitted J/K Matrices <==

    J tasked:                   Yes
    K tasked:                    No
    wK tasked:                   No
    OpenMP threads:               1
    Memory [MiB]:                 2
    Algorithm:                 Core
    Schwarz Cutoff:           1E-12
    Mask sparsity (%):       0.0000
    Fitting Condition:        1E-10

   => Auxiliary Basis Set <=

  Basis Set: (AUG-CC-PVDZ AUX)
    Blend: AUG-CC-PVDZ-JKFIT
    Number of shells: 52
    Number of basis function: 150
    Number of Cartesian functions: 171
    Spherical Harmonics?: true
    Max angular momentum: 3

  Cached 50.0% of DFT collocation blocks in 0.418 [GiB].

   => External Potential Field:  <= 

    > Charges [a.u.] < 

              Z          x          y          z
        2.00000   33.00000   30.00000   30.00000
       -1.00000   25.90000   25.00000   30.00000
       -1.00000   36.10000   35.00000   30.00000
        1.00000   45.90000   30.00000   30.00000
       -1.00000   20.10000   35.00000   30.00000

  Old nuclear repulsion        =    8.794824250261485
  Additional nuclear repulsion =    2.259485448355315
  Total nuclear repulsion      =   11.054309698616800

  Minimum eigenvalue in the overlap matrix is 3.1974438574E-03.
  Using Symmetric Orthogonalization.

  SCF Guess: Superposition of Atomic Densities via on-the-fly atomic UHF.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RKS iter SAD:   -76.00179313507280   -7.60018e+01   0.00000e+00 
   @DF-RKS iter   1:   -76.20910415735386   -2.07311e-01   1.67511e-02 DIIS
   @DF-RKS iter   2:   -74.41270902111802    1.79640e+00   4.31866e-02 DIIS
   @DF-RKS iter   3:   -76.36309435484607   -1.95039e+00   6.16608e-03 DIIS
   @DF-RKS iter   4:   -76.39380736156798   -3.07130e-02   3.33973e-03 DIIS
   @DF-RKS iter   5:   -76.40277633454126   -8.96897e-03   9.56518e-04 DIIS
   @DF-RKS iter   6:   -76.40351968428759   -7.43350e-04   6.34767e-05 DIIS
   @DF-RKS iter   7:   -76.40351779268727    1.89160e-06   7.27743e-05 DIIS
   @DF-RKS iter   8:   -76.40352260041408   -4.80773e-06   6.65602e-06 DIIS
   @DF-RKS iter   9:   -76.40352263847818   -3.80641e-08   4.92134e-07 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1A    -19.088808     2A     -1.207445     3A     -0.771617  
       4A     -0.609041     5A     -0.561082  

    Virtual:                                                              

       6A     -0.476406     7A     -0.197112     8A     -0.160349  
       9A     -0.131770    10A     -0.117031    11A     -0.045653  
      12A      0.001654    13A      0.024232    14A      0.033048  
      15A      0.089265    16A      0.118228    17A      0.164844  
      18A      0.212456    19A      0.329250    20A      0.364700  
      21A      0.448426    22A      0.607404    23A      0.616760  
      24A      0.646773    25A      0.784186    26A      0.860984  
      27A      0.904101    28A      0.996357    29A      1.384864  
      30A      1.414838    31A      1.505430    32A      1.704950  
      33A      1.824563    34A      1.868368    35A      2.078517  
      36A      2.272907    37A      2.901805    38A      2.907463  
      39A      2.927760    40A      3.227069    41A      3.413143  

    Final Occupation by Irrep:
              A 
    DOCC [     5 ]

  @DF-RKS Final Energy:   -76.40352263847818

   => Energetics <=

    Nuclear Repulsion Energy =             11.0543096986167999
    One-Electron Energy =                -123.9236094717930996
    Two-Electron Energy =                  45.5921657573358488
    DFT Exchange-Correlation Energy =      -9.1263886226377338
    Empirical Dispersion Energy =           0.0000000000000000
    VV10 Nonlocal Energy =                  0.0000000000000000
    Total Energy =                        -76.4035226384781794

Computation Completed


Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]

Properties computed using the SCF density matrix

  Nuclear Dipole Moment: [e a0]
     X:   572.0806      Y:   570.7748      Z:   571.3417

  Electronic Dipole Moment: [e a0]
     X:  -573.5381      Y:  -571.1430      Z:  -570.8976

  Dipole Moment: [e a0]
     X:    -1.4576      Y:    -0.3682      Z:     0.4441     Total:     1.5676

  Dipole Moment: [D]
     X:    -3.7047      Y:    -0.9359      Z:     1.1288     Total:     3.9844


*** tstop() called on rich133-k41-16-l.pace.gatech.edu at Mon Jan 27 08:41:53 2020
Module time:
	user time   =      15.70 seconds =       0.26 minutes
	system time =       0.78 seconds =       0.01 minutes
	total time  =         18 seconds =       0.30 minutes
Total time:
	user time   =      15.70 seconds =       0.26 minutes
	system time =       0.78 seconds =       0.01 minutes
	total time  =         18 seconds =       0.30 minutes

    Psi4 stopped on: Monday, 27 January 2020 08:41AM
    Psi4 wall time for execution: 0:00:17.83

*** Psi4 exiting successfully. Buy a developer a beer!
